tap_zendesk/streams.py

```diff
@@ -203,13 +206,9 @@ def sync(self, state):
             # If we make it here, all quality checks have passed. Reset retry count.
             num_retries = 0
```

Comment:
`num_retries` is always = 0, can we just remove it?

Reply:
The increment is hidden in this PR